integration_test/README.md

```diff
@@ -13,13 +14,16 @@ This will build docker image `amoc:latest`.
 ### 2. Run docker containers
 `./integration_test/test_docker_image.sh`
 
-This will start two containers `amoc-1` and `amoc-2` and will wait until healthcheck is successful.
+This will start two containers `amoc-1` and `amoc-2` and
+will wait until healthcheck is successful.
 
 ### 3. Test installation of user scenario in amoc cluster
 
- `./integration_test/test_distribute_scenario.sh`
+`./integration_test/test_distribute_scenario.sh`
  
-This will install a sample `dummy_scenario.erl` on node `amoc-1` using curl and then verify that it was propagated to the other node in the cluster `amoc-2`.
+This will install a sample `dummy_scenario.erl` on node `amoc-1`
+using curl and then verify that it was propagated to the other
+node in the cluster `amoc-2`.
```

Comment:
```suggestion
This will install a sample `dummy_scenario.erl` on the `amoc-1` node
using curl and then it will verify that it was propagated to the other
node in the cluster, `amoc-2`.
```